api/schemas/webhook-get.json

```diff
@@ -0,0 +1,33 @@
+{
+    "title": "Webhook Detail",
+    "description": "Describes a Webhook",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "webhook-with-id.json"
+        },
+        {
+            "type": "object",
+            "required":
+            [
+                "status"
+            ],
+            "properties": {
+                "error": {
+                    "description": "Provides more information for the error status, as described by the [Error](../schemas/error#top) type",
+                    "$ref": "error.json"
+                },
+                "status": {
```

Comment:
Sense check: are we happy this strikes the right balance of simplicity versus richness? The GitHub webhook experience is probably too far towards "richness" here: it shows you recent deliveries and their payloads and headers etc. Should we consider adding "last delivery" or "successful delivery count" or generally transforming the `error` key into more of a status? Or is this enough, and the store logs should have the rest?

Reply:
This approach was taken to match the pattern used on the delete requests endpoint. The addition being that we're adding the ability to temporarily disable the webhooks, and to "re-set" them on an error condition via the `status` param. The problem with relying on store logs is that most clients won't have access to them. I'll defer to @danjhd for thoughts on "last delivery"/"successful delivery count" though.

Reply:
Yes, I think that makes sense, and as you point out, adding more is easy enough

Reply:
Ok, I'm on board with all of that, thanks both!